### PR TITLE

Fix error format based on best practices from 


### PR DESCRIPTION
**Pull request by CodelingoBotTester**
Hi Daanikus and team!

Dan here from CodeLingo. You can use [CodeLingo](https://codelingo.io) to automatically fix function comments following the
[Effective Go guidelines](https://golang.org/doc/effective_go.html#commentary) in your contributing doc.

This patch was generated by running the CodeLingo Rewrite Flow over the "[go-error-fmt](https://github.com/codelingo/codelingo/blob/master/tenets/codelingo/code-review-comments/go-error-fmt/codelingo.yaml)" Tenet. Note: the same Tenet can be used to automate PR reviews and generate contributor docs by [Installing](https://github.com/apps/codelingo) the CodeLingo GitHub app. Learn more at [codelingo.io](https://codelingo.io).

### CODE REVIEW

**Comment by Daanikus** [comment]
this is a test message

**Comment by Daanikus** [comment]
this is a test message

**Comment by Daanikus** [comment]
this is another test message

**Comment by Daanikus** [comment]
this is a test message

**Comment by Daanikus** [comment]
this is another test message

**Comment by Daanikus** [comment]
this is a test message

**Comment by Daanikus** [comment]
this is another test message

**Comment by Daanikus** [comment]
this is a test message

**Comment by Daanikus** [comment]
this is another test message

**Comment by Daanikus** [comment]
Message to be added

File: new.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function(name) {
```
Comment (CodelingoBotTester) [inline new.js]:
```diff
- // NoComment is a function
+ // NoComment {{ .commentVar }}
```
Change `commentVar` by commenting: `@CodeLingoBot rewrite set commentVar "your comment"`

**Pull request closed without merging**